java/java.lsp.server/vscode/package.json

```diff
@@ -4,7 +4,7 @@
 	"description": "Apache NetBeans Language Server Extension for Visual Studio Code",
 	"author": "Apache NetBeans",
 	"license": "Apache 2.0",
-	"version": "12.3.0",
+	"version": "12.3.9",
```

Comment:
What does this version means?

Reply:
This means 12.4 Beta as it was discussed on dev@nb mailing list. It is shorter than proposed 12.3.990 yet still will serve the purpose.

Reply:
The last version before 12.4.0 ? Hard to assign a reasonable version number for 12.4 Beta...